apps/src/templates/PauseButton.jsx

```diff
@@ -60,28 +66,29 @@ class PauseButton extends React.Component {
 const styles = {
   icon: {
     lineHeight: 'inherit',
-    color: color.white
+    fontSize: '48px'
   },
   container: {
     width: 40,
     height: 40,
     lineHeight: '40px',
-    textAlign: 'center',
-    verticalAlign: 'middle',
-    display: 'inline-block'
+    display: 'flex',
+    justifyContent: 'center'
   },
   button: {
     minWidth: 0,
     padding: 0,
-    borderRadius: '100%'
+    borderRadius: '100%',
+    backgroundColor: color.white
   },
-  runningColor: {
-    backgroundColor: color.cyan,
-    borderColor: color.cyan
+  inactiveColor: {
+    color: '#C7C7C7'
```

Comment:
It seems like this might be worth centralizing at some point, but okay for later.